tests/nnfw_api/src/GenModelTests.h

```diff
@@ -0,0 +1,108 @@
+/*
```

Comment:
As the class name is `GenModelTest` so what about renaming this file to `GenModelTest.h`?